web/Areas/Directory/Views/Index.cshtml

```diff
@@ -0,0 +1,151 @@
+            }
+        </q-td>
+    </template>
+    <template v-slot:body-cell-links="props">
+        <q-td :props="props">
+            @if (UserHelper.HasPermission(rapsContext, UserHelper.GetCurrentUser(), "SVMSecure.userinfo"))
+            {
+                @: <q-btn :props="props" size="sm" padding="xs" color="primary" square flat icon="person" :href="'userInfo/' + props.row.mothraId"></q-btn>
+            }
+            @if (UserHelper.HasPermission(rapsContext, UserHelper.GetCurrentUser(), "SVMSecure.SU"))
+            {
+                @: <q-btn :props="props" size="sm" padding="xs" color="primary" square flat icon="face" :href="'/EmulateUser/' + props.row.loginId"></q-btn>
+            }
+        </q-td>
+    </template>
+</q-table>
+@section Scripts {
+    <script src="~/js/qtable.js"></script>
+    <script asp-add-nonce="true">
+        createVueApp({
+            data() {
+                return {
+                    userSearch: getItemFromStorage("searchField") ?? "",
+                    users: new quasarTable({
+                        keys: ["mothraId"],
+                        urlBase: "users",
+                        columns: [
+                            { name: "photo", label: "", field: "", align: "center", style: "width:87px; height:111px;" },
+                            { name: "name", label: "Individual", field: "name", align: "left", style: "max-width:10em; text-wrap:wrap;", sortable: true },
+                            { name: "title", label: "Title", field: "title", align: "left", style: "max-width:10em; text-wrap:wrap;", sortable: true },
+                            { name: "department", label: "Department", field: "department", style: "max-width:10em; text-wrap:wrap;", align: "left", sortable: true },
+                            { name: "ids", label: "", field: "", align: "left" },
+                            @if (UserHelper.HasPermission(rapsContext, UserHelper.GetCurrentUser(), "SVMSecure.DirectoryDetail")) {
+                                @: { name: "detail", label: "", field: "", align: "left" },
+                            }
+                            { name: "loginid", label: "Login ID", field: "loginId", align: "left", sortable: true },
+                            { name: "email", label: "Email", field: "mailId", align: "left", sortable: true },
+                            { name: "links", label: "", field: "", align: "left", style: "width:50px;" }
+                        ],
+                        pagination: { rowsPerPage: 0 }
+                    })
+                }
+            },
+            methods: {
+                findUsers: async function () {
+                    this.users.urlBase = "search/" + this.userSearch
+                    if (this.userSearch.length >= 3) {
```

Comment:
I know I used 3 as the min length for RAPS, but it's been suggested to me that 2 might be better.